content/blog/2023-05-02-response-to-bitcoin-magazine-article-about-tokenization/index.md

```diff
@@ -0,0 +1,124 @@
+
+Proof of work is highlighted above because it is the only true consensus mechanism that makes a blockchain truly decentralized, therefore secure.
+
+### Mr. Koss Wrote:
+
+*“Tokenizing assets like real estate will only make matters worse as crowds shove money into the market, driving prices higher. It becomes a self-fulfilling prophecy. People buy houses because they know the prices will go up, then the prices do go up and more demand comes in to chase the gains. Every investor following their individual incentives puts owning a home further out of reach for the average citizen. This is not a solution.”*
+
+### Our response:
+
+It is not possible to make matters worse by improving the property records system at the national, provincial and municipal levels. Better property systems make better markets and thus better access and prices to the general public.
+
+Indeed, tokenization will likely accelerate the price deflation of real estate assets and homes. This will not only happen because people will save in BTC and ETC, draining home premiums, but because the uncertainty of legal property itself will be removed.
+
+That people are now experimenting with the technology, playing with NFTs, speculating, scamming, and losing money is just the prelude and the training wheels phase of the system. All these activities are only serving as a learning process and showing the abilities of the technology. 
+
+### Mr. Koss Wrote:
+
+*“Furthermore, a “blockchain” is simply a ledger, or record of who owns what. With Ethereum especially, there is no meaningful link to the real world which would allow for native contract enforcement, preventing rug pulls of these token holders. The whole system ultimately relies upon legacy law enforcement and the judicial system to uphold the property rights of these investors — a system that appears to be increasingly hostile in enforcement actions against the crypto industry writ large.”*
+
+### Our response:
+
+Exactly! A programmable blockchain is a ledger with a record of who owns what, but in the case of ETC (not Ethereum), incredibly secure. 
+
+There is no meaningful link between current systems and physical assets either! We would argue the current systems are even less linked than if assets were represented on the blockchain.
+
+Yes, the whole system will always be dependent on laws and their enforcement. This will never be delinked even for onchain assets as BTC and ETC. You may have 1 BTC or 100 ETC in the account you control through your private key, but that does not mean that the asset is your property. Legal property is not the same as physical access and control. The latter is what blockchains guarantee, not the former. 
```

Comment:
Line 90: Add hyphen; Correctr quantifier usager
```suggestion
Yes, the whole system will always be dependent on laws and their enforcement. This will never be delinked even for on-chain assets such as BTC and ETC. You may have 1 BTC or 100 ETC in the account you control through your private key, but that does not mean that the asset is your property. Legal property is not the same as physical access and control. The latter is what blockchains guarantee, not the former. 
```